File: python/exercises/exercise 06.py
import os
os.system("cls||clear")

soma = 0

for i in range(1,6):
    numeros = int(input("Digite um número inteiro: "))
    soma += numeros
    
for i in range(1,6):
    print(f"{i}º número inteiro: {numeros}")

print(f"\nSoma: {soma}")

File: c/exercises/exercise06.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(){
	setlocale(LC_ALL, "");
	
	float valor, reajuste = 0.9, valorFinal;
	
	printf("Digite um número: ");
	scanf("%f",&valor);
    
    valorFinal = valor * reajuste;
    
	system("cls||clear");
	printf("=== Exibindo resultados ===\n");
	printf("Valor: %.2f \n", valor);
	printf("Valor com desconto: %.2f \n", valorFinal);
	
	return 0;
}
